com.stremio.Stremio.json

```diff
@@ -0,0 +1,204 @@
+{
+  "app-id": "com.stremio.Stremio",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.12",
+  "sdk": "org.kde.Sdk",
+  "command": "/app/opt/stremio/stremio",
+  "separate-locales": true,
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--device=dri",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--talk-name=org.a11y.Bus",
+    "--talk-name=org.freedesktop.ScreenSaver",
+    "--talk-name=org.freedesktop.Notifications",
+    "--talk-name=org.kde.StatusNotifierWatcher",
+    "--own-name=org.kde.*",
+    "--env=TMPDIR=/var/cache/tmp"
+  ],
+  "cleanup": [
+    "/include",
+    "/lib/pkgconfig",
+    "/share/man",
+    "/bin/python"
+  ],
+  "modules": [
+    {
+      "name": "nodejs",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -D bin/node /app/bin/node"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://nodejs.org/dist/latest-v8.x/node-v8.17.0-linux-x64.tar.gz",
+          "sha256": "8b2c9e1f84317c4b02736c4c50db4dd2cd6c4f0ba910fa81f887c8c9294af596"
+        }
+      ]
+    },
+    {
+      "name": "stremio",
+      "buildsystem": "qmake",
+      "build-options": {
+        "env": {
+          "LIBRARY_PATH": "/app/lib:/usr/lib"
+        }
+      },
+      "post-install": [
+        "install -Dm755 server.js /app/opt/stremio/server.js",
+        "install -Dm644 com.stremio.Stremio.desktop /app/share/applications/com.stremio.Stremio.desktop",
+        "install -Dm644 ./images/stremio.svg /app/share/icons/hicolor/scalable/apps/com.stremio.Stremio.svg",
+        "install -Dm644 ./images/stremio_tray_black.svg /app/share/icons/hicolor/scalable/apps/com.stremio.Stremio-symbolic.svg",
+        "install -Dm644 com.stremio.Stremio.appdata.xml /app/share/metainfo/com.stremio.Stremio.appdata.xml",
+        "ln -s /app/bin/node /app/opt/stremio/node"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/Stremio/stremio-shell.git",
+          "tag": "v4.4.120"
+        },
+        {
+          "type": "file",
+          "url": "https://dl.strem.io/four/v4.4.120/server.js",
+          "sha256": "241f5633528e9f19c84c7ca41266771c309e286d6d9393a99e0fbce2203a3caa"
+        },
+        {
+          "type": "file",
+          "path": "com.stremio.Stremio.desktop"
+        },
+        {
+          "type": "file",
+          "path": "com.stremio.Stremio.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
Fixed it